styleguide/toolbar.scss

```diff
@@ -14,7 +14,7 @@ $height: 48px;
 }
 
 .kiln-toolbar {
-  background-color: $black-60;
+  background-color: $toolbar-view;
```

Comment:
Depends if you want the lighter toolbar now or want to wait until the Edit/View toggle is in place.
I chose to leave the darker grey for now.